docs/en/operation/Journal.md

```diff
@@ -72,7 +72,7 @@ backup directory can be configured by setting `alluxio.master.backup.directory`
 alluxio.master.backup.directory=/alluxio/backups
 ```
 
-See the [backup command documentation]({{ '/en/operation/Admin-CLI.html' | relativize_url }}#backup)
+See the [backup command documentation]({{ '/en/operation/Admin-CLI.html#backup' | relativize_url }}#backup)
```

Comment:
why two `#backup`?

Reply:
my fault. I just copied the enterprise PR without looking. Fixed